Src/BootCamp.Chapter/TextTable.cs

```diff
@@ -34,7 +38,51 @@ public static class TextTable
         /// </summary>
         public static string Build(string message, int padding)
         {
+            if (!string.IsNullOrEmpty(message))
+            {
+                var sb = new StringBuilder();
+                var messages = message.Split(Environment.NewLine);
+                var longestString = "";
+                for (int i = 0; i < messages.Length; i++)
+                {
+                    if (messages[i].Length > longestString.Length)
+                        longestString = messages[i];
+                }
+                var whiteSpace = "";
+                var length = longestString.Length + padding * 2;
+                sb.Append($"+{whiteSpace.PadRight(length, '-')}+{Environment.NewLine}");
```

Comment:
use String.Empty instead your variable.

Reply:
removed whiteSpace variable and have used String.Empty